lib/Runtime/Library/InJavascript/Intl.js

```diff
@@ -78,6 +78,15 @@
         return defaultLocale;
     }
 
+    let getDefaultTimeZone = function () {
+        let defaultTimeZone = platform.getDefaultTimeZone();
+        if (defaultTimeZone === null) {
```

Comment:
>defaultTimeZone [](start = 12, length = 15)

If GetDefaultTimeZoneId fails in EntryIntl_GetDefaultTimeZone undefined is returned.

Since valid return value is a string why not check for typeof?

Reply:
A return of `undefined` was, in general, accounted for in the Intl.js logic for WinGlob. I added returns of null for unimplemented platform APIs and the fallback behavior was meant to catch that. I imagine that the fallback paths could be generalized to any implementation returning something other than the expected value. Will take note of this for a refactoring later on. Thanks!